data-exporter/src/test/java/org/reactome/release/dataexport/ReactomeEventTest.java

```diff
@@ -34,7 +33,7 @@ public void retrievesNCBIEntryFromDummyGraphDB() {
 		Map<Long, ReactomeEvent> reactomeEventCache =
 			ReactomeEvent.fetchReactomeEventMap(dummyGraphDBServer.getSession());
 
-		assertThat(reactomeEventCache, is(aMapWithSize(7)));
+		assertThat(reactomeEventCache, is(aMapWithSize(15)));
```

Comment:
I guess this is one of the magic numbers you'd want to replace with a properly named variable?

Reply:
Yes, this was the one I was thinking about yesterday.  I will replace it with a named variable!